Exercise_2.py

```diff
@@ -1,23 +1,42 @@
 # Python program for implementation of Quicksort Sort 
   
 # give you explanation for the approach
+#Time complexity O(n log(n)) - best case, O(n^2) - worst case
+#Space complexity O(1) - in-place
```

Comment:
There will be recursive stack space used under the hood. The SC should be O(log(n))